fiftyone/utils/labelstudio.py

```diff
@@ -0,0 +1,835 @@
+
+    Returns:
+        A tuple of parent tag, child tag and parent_tag kwargs
+    """
+    x = _LABEL_TYPES.get(
+        label_type, _LABEL_TYPES[foua._RETURN_TYPES_MAP[label_type]]
+    )
+    return x["parent_tag"], x["child_tag"], x.get("tag_kwargs", {})
+
+
+def _label_class_from_tag(label_type: str):
+    """Map Label Studio parent tag to fo.Label"""
+    reverse = {
+        v["parent_tag"].lower(): v["label"] for v in _LABEL_TYPES.values()
+    }
+    if label_type in reverse:
+        return reverse[label_type]
+    raise ValueError(f"Unknown label type: {label_type}")
+
+
+def _tag_from_label(label_cls):
+    label_cls = label_cls.lower()
+    if label_cls in _LABEL2TYPE:
+        label_type = _LABEL2TYPE[label_cls]
+        return _LABEL_TYPES[label_type]["parent_tag"]
+    if label_cls in foua._RETURN_TYPES_MAP:
+        label_type = foua._RETURN_TYPES_MAP[label_cls]
+        return _LABEL_TYPES[label_type]["parent_tag"]
+    else:
+        raise ValueError(f"Unsupported label class: {label_cls}")
+
+
+def _normalize_values(values: List[float]) -> List[float]:
+    values = np.array(values) / 100
+    return values.tolist()
+
+
+def _denormalize_values(values: List[float]) -> List[float]:
+    values = np.array(values) * 100
+    return values.tolist()
+
+
+def _get_label_ids(label):
+    if isinstance(
+        label, (fol.Classification, fol.Segmentation, fol.Regression)
+    ):
+        ids = label.id
+    elif isinstance(label, fol.Classifications):
+        ids = [l.id for l in label.classifications]
+    elif isinstance(label, fol.Detections):
+        ids = [l.id for l in label.detections]
+    elif isinstance(label, fol.Polylines):
+        ids = [l.id for l in label.polylines]
+    elif isinstance(label, fol.Keypoints):
+        ids = [l.id for l in label.keypoints]
+    else:
+        raise ValueError(f"Unsupported label type: {type(label)}")
+    return ids
+
+
+_LABEL_TYPES = {
+    "classification": dict(
+        parent_tag="Choices",
+        child_tag="Choice",
+        label=fol.Classification,
+        tag_kwargs={"choice": "single-radio"},
+    ),
+    "classifications": dict(
+        parent_tag="Choices",
+        child_tag="Choice",
+        label=fol.Classification,
+        tag_kwargs={"choice": "multiple"},
+    ),
+    "detections": dict(
+        parent_tag="RectangleLabels",
+        child_tag="Label",
+        label=fol.Detection,
+    ),
+    "polylines": dict(
+        parent_tag="PolygonLabels",
+        child_tag="Label",
+        label=fol.Polyline,
+    ),
+    "keypoints": dict(
+        parent_tag="KeyPointLabels",
+        child_tag="Label",
+        label=fol.Keypoint,
+    ),
+    "segmentation": dict(
+        parent_tag="BrushLabels",
+        child_tag="Label",
+        label=fol.Segmentation,
+    ),
+    "scalar": dict(
+        parent_tag="Number",
+        child_tag=None,
+        label=fol.Regression,
+    ),
+}
+_LABEL2TYPE = {v["label"].__name__: k for k, v in _LABEL_TYPES.items()}
```

Comment:
Nit: Generally we prefer to explicitly type out variable names.
```suggestion
_LABEL_TO_TYPE = {v["label"].__name__: k for k, v in _LABEL_TYPES.items()}
```